.circleci/config.yml

```diff
@@ -684,10 +694,18 @@ step-maybe-restore-git-cache: &step-maybe-restore-git-cache
     paths:
       - ~/.gclient-cache
     keys:
-      - v2-gclient-cache-{{ arch }}-{{ checksum "src/electron/.circle-sync-done" }}-{{ checksum "src/electron/DEPS" }}
-      - v2-gclient-cache-{{ arch }}-{{ checksum "src/electron/.circle-sync-done" }}
+      - v2-gclient-cache-{{ checksum "src/electron/.circle-sync-done" }}-{{ checksum "src/electron/DEPS" }}
```

Comment:
why is the arch no longer part of the cache key?

Reply:
Because it's just a git blob, not arch specific.  Also we checkout macos on a linux machine anyway 🤷‍♂